### PR TITLE
[vm] Remove old account create function

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

To deprecate the old implementation of create account in the LibraAccount module. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### CODE REVIEW

**Comment by bob-wilson** [comment]
The terminology doesn't seem quite right here. "Deprecate" means to keep something around but discourage people from using it. This looks like it is removing the old function. If that's right, can you retitle the PR and the git commit message?

File: language/vm/vm-runtime/src/interpreter.rs [hunk 1]
```diff
@@ -62,7 +62,7 @@ lazy_static! {
 
 // Names for special functions and structs
 lazy_static! {
-    static ref CREATE_ACCOUNT_NAME: Identifier = Identifier::new("make").unwrap();
+    static ref CREATE_ACCOUNT_NAME: Identifier = Identifier::new("create_account").unwrap();
```
Comment (sblackshear) [inline language/vm/vm-runtime/src/interpreter.rs]: I also see a `CREATE_ACCOUNT_NAME` in `txn_executor.rs` that needs to be updated. In fact, several of these magic names are duplicated there. Should we unify these lists?

Comment (runtian-zhou) [reply language/vm/vm-runtime/src/interpreter.rs]: I totally agree that we should unify these list of special names. I'll address this issue during the MoveVM refactor.

**Comment by davidiw** [comment]
Please re-base and re-request if still valid.

**Pull request closed without merging**